### PR TITLE
47370_supervisor_add_new_course

### PR DESCRIPTION
**Pull request by asifbuetcse**
Due to technical difficulties with internet; we can't access the bootstrap CDN. That's why I have downloaded them to my project and used them locally.

### CODE REVIEW

**Comment by yasin-buet** [comment]
ok

File: resources/views/layouts/app.blade.php [hunk 1]
```diff
@@ -1,10 +1,10 @@
 <html>
 <head>
     <title>tms_42</title>
-    <link rel="stylesheet" href="http://maxcdn.bootstrapcdn.com/bootstrap/3.3.5/css/bootstrap.min.css">
-    <script src="https://ajax.googleapis.com/ajax/libs/jquery/1.11.3/jquery.min.js"></script>
-    <script src="http://maxcdn.bootstrapcdn.com/bootstrap/3.3.5/js/bootstrap.min.js"></script>
     {!! Html::style('css/stylesheet.css') !!}
+    {!! Html::style('bootstrap.min.css') !!}
```
Comment (dinhhoanglong91) [inline resources/views/layouts/app.blade.php]: Move all css to css folder and js to js folder.

File: app/Http/Controllers/Supervisor/CoursesController.php [hunk 2]
```diff
@@ -40,7 +42,9 @@ public function create()
      */
     public function store(Request $request)
     {
-        //
+        $input = $request->all();
+        Course::create($input)->subjects()->attach($request->input('subjects'));
```
Comment (dinhhoanglong91) [inline app/Http/Controllers/Supervisor/CoursesController.php]: Excellent !
It will be better if you validate input data.

**Comment by dinhhoanglong91** [comment]
ok

**Pull request merged**